voidcrew/modules/cloning/code/machinery_cloning.dm

```diff
@@ -0,0 +1,572 @@
+//Cloning revival method.
+//The pod handles the actual cloning while the computer manages the clone profiles
+
+//Potential replacement for genetics revives or something I dunno (?)
+
+#define CLONE_INITIAL_DAMAGE     150    //Clones in clonepods start with 150 cloneloss damage and 150 brainloss damage, thats just logical
+#define MINIMUM_HEAL_LEVEL 40
+
+#define SPEAK(message) radio.talk_into(src, message, radio_channel)
+
+/obj/machinery/clonepod
+	name = "cloning pod"
+	desc = "An electronically-lockable pod for growing organic tissue."
+	density = TRUE
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_0"
+	req_access = list(ACCESS_CLONING) //FOR PREMATURE UNLOCKING.
+	verb_say = "states"
+	circuit = /obj/item/circuitboard/machine/clonepod
+
+	var/heal_level //The clone is released once its health reaches this level.
+	var/obj/machinery/computer/cloning/connected //So we remember the connected clone machine.
+	var/mess = FALSE //Need to clean out it if it's full of exploded clone.
+	var/attempting = FALSE //One clone attempt at a time thanks
+	var/speed_coeff
+	var/efficiency
+
+	var/datum/mind/clonemind
+	var/grab_ghost_when = CLONER_MATURE_CLONE
+
+	var/internal_radio = TRUE
+	var/obj/item/radio/radio
+	var/radio_key = /obj/item/encryptionkey/headset_med
+	var/radio_channel = RADIO_CHANNEL_MEDICAL
```

Comment:
we don't plan on having departmental radios so it's probably fine to gut stuff related to it